### PR TITLE
feat: Add a select all checkbox on the  moonraker backup and restore dialogs 

### LINKED ISSUE #1260: [FR] [QOL] 'Select All' checkbox on Moonraker Database Backup/Restore UIs
### Requested feature:

Low Priority Feature Request:  Addition of a 'Select All' checkbox at the top of the top of the Moonraker Database Backup/Restore UIs, defaulting to all elements being selected.

### Solves the following problem:

For those building/rebuilding/commissioning multiple Klipper/Mainsail based printers, this will provide a QOL improvement.

### Additional information:

_No response_

### PR DESCRIPTION
**Pull request by FGRCL**
## Description
This PR adds a two checkboxes to select all the checkbox in the backup and restore dialogs. Under Settings->General->Moonraker DB

## Related Tickets & Documents
Fixes #1260 

## Mobile & Desktop Screenshots/Recordings
backup
restore

## [optional] Are there any post-deployment tasks we need to perform?
No

Signed-off-by: François LaBerge FGRCLaBerge@gmail.com

### CODE REVIEW

File: src/components/inputs/CheckboxList.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<template>
+    <v-col class="pl-6">
+        <template v-if="selectAll">
+            <v-checkbox
+                v-model="selectAllModel"
+                :label="$t('Settings.GeneralTab.Everything')"
```
Comment (meteyou) [inline src/components/inputs/CheckboxList.vue:6]: pls extract the label of "everything" in a prop

File: src/components/inputs/CheckboxList.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    setSelectAll(state: boolean) {
+        if (state) {
+            this.selectedCheckboxes = this.options.map((o) => o.value)
+        } else {
+            this.selectedCheckboxes = []
+        }
+    }
+
+    mounted() {
+        console.log(this.selectAll)
```
Comment (meteyou) [inline src/components/inputs/CheckboxList.vue:65]: pls remove this debug output

**Comment by meteyou** [comment]
Thx for your PR. The function looks good to me. I only found 2 tiny issues. Please fix these and this PR is merge ready.

**Comment by FGRCL** [comment]
Done!

File: src/components/inputs/CheckboxList.vue (hunk 2 above)
Comment (FGRCL) [reply src/components/inputs/CheckboxList.vue]: Oops, should'be reviewed that more carefully.

**Review by meteyou — APPROVED** [review]

**Review by rackrick — APPROVED** [review]

**Comment by meteyou** [comment]
@FGRCL thank you very much for contributing!

**Pull request merged**